src/shared/models/game/Market.es6

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+//
+// Market
+// Created by dpekar on 10/8/15.
+//
+
+const BaseModel = require('../BaseModel')
+const Tile = require('./Tile')
+
+class Market extends BaseModel {
+    constructor(tilePiles) {
+        super()
+        this.set('tiles', [])
+        this.fillUpSlots(tilePiles)
+    }
+
+    getTiles() { return this.get('tiles') }
+
+
+    fillUpSlots(tilePiles) {
```

Comment:
i assume this class will also have a `replenishMarket`, or some similar method for adding a single tile? also `trashTileInMarket`